### DISCUSSION TITLE
poetry can't find a version, when allowing "*"

### DISCUSSION THREAD

**Discussion by eike-fokken** [casadi/casadi#3529] [Q&A] [answered]
Hi!

When allowing every version of casadi in the pyproject.toml:

```
[tool.poetry.dependencies]
python = "^3.10" 
casadi = "*"
```

I get this error message:

```
Updating dependencies
Resolving dependencies... (10.0s)

Package operations: 0 installs, 1 update, 0 removals

  • Updating casadi (3.6.4 -> 3.6.4.post1): Failed

  RuntimeError

  Unable to find installation candidates for casadi (3.6.4.post1)

  at ~/.pyenv/versions/3.10.13/lib/python3.10/site-packages/poetry/installation/chooser.py:73 in choose_for
       69│
       70│             links.append(link)
       71│
       72│         if not links:
    →  73│             raise RuntimeError(f"Unable to find installation candidates for {package}")
       74│
       75│         # Get the best link
       76│         chosen = max(links, key=lambda link: self._sort_key(package, link))
       77│

Cannot install casadi.
```

when replacing

```
casadi = "*"
```

by 

```
casadi = "3.6.4"
```

everything works. Is this a bug in casadi 3.6.4.post1, or somewhere else or is there some configuration I can set to make it work without fixing the casadi version?

**Comment by jgillis** [comment]
casadi 3.6.4.post1 was an attempt to update some meta-data over at pypi.

I'm not sure what the correct course of action would be. Deleting that release may fix the issue, or make things even worse..

Feel free to share any tips

At least `pip install casadi` shows no issue for me.

**Comment by jgillis** [comment, marked as answer]
I tried the 'yank' feature of pypi. Can you check if this solves the issue?

**Comment by eike-fokken** [comment]
It worked, thanks! Unfortunately I also don't know, what the best course of action is. I also don't know, where the problem lies. Maybe poetry has a bug, but I didn't find one that looked related.
